amplitude-experiment.gemspec

```diff
@@ -14,7 +14,9 @@ Gem::Specification.new do |spec|
   spec.files                 = Dir['README.md',
                                    'lib/**/*.rb',
                                    'experiment-ruby.gemspec',
-                                   'Gemfile']
+                                   'Gemfile',
+                                   'lib/experiment/local/evaluation/lib/**/*.so',
+                                   'lib/experiment/local/evaluation/lib/**/*.dylib']
```

Comment:
I've validated that the gem works again by installing a local version, but I only have an intel macbook to test this on, so I can't verify this works for all architectures